ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/disconnected/workers-rhel8/provision/cucushift-installer-rehearse-gcp-ipi-disconnected-workers-rhel8-provision-chain.yaml

```diff
@@ -0,0 +1,34 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-disconnected-workers-rhel8-provision
+  steps: 
+  - chain: cucushift-installer-rehearse-gcp-ipi-disconnected-provision
+  - ref: workers-rhel-gcp-provision
+  - ref: workers-rhel-pre-hook-sshtunnel
+  - chain: workers-rhel
+  - ref: ipi-install-times-collection
```

Comment:
This ref is already included in cucushift-installer-rehearse-gcp-ipi-disconnected-provision, so need it here.